images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -318,32 +323,47 @@ function Get-PacmanVersion {
     $rawVersion = & $pacmanPath --version
     $rawVersion.Split([System.Environment]::NewLine)[1] -match "\d+\.\d+(\.\d+)?" | Out-Null
     $pacmanVersion = $matches[0]
-    return "Pacman $pacmanVersion"
+    return $pacmanVersion
 }
 
 function Get-YAMLLintVersion {
-    yamllint --version
+    return ((yamllint --version) -replace "^yamllint").Trim()
 }
 
 function Get-BizTalkVersion {
     $bizTalkReg = Get-ItemProperty "HKLM:\SOFTWARE\WOW6432Node\Microsoft\BizTalk Server\3.0"
-    return "$($bizTalkReg.ProductName) $($bizTalkReg.ProductVersion)"
+    return [ToolVersionNode]::new($($bizTalkReg.ProductName), $($bizTalkReg.ProductVersion))
 }
 
 function Get-PipxVersion {
     $pipxVersion = pipx --version
-    return "Pipx $pipxVersion"
+    return $pipxVersion
 }
 
 function Build-PackageManagementEnvironmentTable {
-    return @(
-        [PSCustomObject] @{
+    $node = [HeaderNode]::new("Environment variables")
+    $envVariables = @(
+        @{
             "Name" = "VCPKG_INSTALLATION_ROOT"
             "Value" = $env:VCPKG_INSTALLATION_ROOT
-        },
-        [PSCustomObject] @{
-            "Name" = "CONDA"
-            "Value" = $env:CONDA
         }
     )
+    if (Test-IsWin19) {
```

Comment:
We shouldn't have this condition at all.
I could be wrong but I think you have outdated version of code in your fork because I have removed this condition a week ago.

Reply:
yes you're right, unfortunately I managed to mix it up with some outdated code parts :(
corrected